pkg/hardware/config.go

```diff
@@ -22,6 +22,13 @@ type HardwareConfig struct {
 	Secrets   []corev1.Secret
 }
 
+type BmcSecretConfig struct {
+	Host     string
+	Username string
+	Password string
+	Vendor   string
+}
```

Comment:
wouldn't this struct make more sense in pbnj's client.go since it's being used in that file but not in this?

Reply:
I agree. This is actually only used as a function parameter input for `ValidateBMCSecretCreds` so it can be in the pbnj package.